src/components/Article.jsx

```diff
@@ -118,13 +122,41 @@ function handleDelete (e) {
     })
 }
 
-function hanldeVoteIncrease (e) {
-    console.log(Number(e.target.value) + 1)
-    patchVotes(articleId).then((res) => {
-        console.log(res)
+function handleVoteIncrease (e) {
```

Comment:
We could try and make this code more DRY, currently it is quite repetitive.

Reply:
Done